### ISSUE TITLE
Exception when printing formulas

### ISSUE THREAD

**Issue by cacfd3a** [jgonggrijp/net-prove#4]
When printing formulas in the ghci console, I am confronted with the following:

```
Exception: <stdout>: hPutChar: invalid argument (invalid character). 
```

As soon as the superscript + and - should be rendered. 

I think we print a little _too_ pretty. Maybe this is a Windows specific problem?

**Comment by jgonggrijp** [comment]
Yes, this might have something to do with your console not (fully) supporting unicode. It works fine on mac and linux. Regardless, maybe we should do something about it.
